### ISSUE TITLE
getPullRequests error with multiple PR types

### ISSUE THREAD

**Issue by unthingable** [codacy/bitbucket-scala-client#23]
https://github.com/codacy/bitbucket-scala-client/blob/master/src/main/scala/com/codacy/client/bitbucket/service/PullRequestServices.scala#L15

Apparently, with the current API listing multiple PR types with a comma leads to a 401. But this works:

```scala
   def getPullRequests(owner: String, repository: String, states: Seq[String] = Seq("OPEN")): RequestResponse[Seq[PullRequest]] = {
    val url = s"https://bitbucket.org/!api/2.0/repositories/$owner/$repository/pullrequests?state=${states.mkString("&state=")}"

    client.executePaginated(Request(url, classOf[Seq[PullRequest]]))
  }
```

From [API docs](https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/pullrequests):

```
state	string	

Only return pull requests that are in this state. This parameter can be repeated.
```

**Comment by pedrorijo91** [comment]
Hey @unthingable 

Thanks for reporting this! Would you like to contribute through a PR? It seems you already found the solution, so it should be very simple

**Comment by unthingable** [comment]
Done!

**Comment by rtfpessoa** [comment]
Released as version 4.0.1. Should hit the repos soon.
